app/index.js

```diff
@@ -41,29 +42,6 @@ var BluemanGenerator = yeoman.generators.Base.extend({
     }, {
       name: 'projectURL',
       message: 'What will the subdomain for your project be? ' + chalk.magenta('(do not include mybluemix.net)')
-    }, {
```

Comment:
Thanks for making these updates, @dzaharia1!!

The only thing I'm a little unsure about is why you removed the memory allocation step. I think it was a nice feature. Add it back in and I'm all about this PR :smile: